awx/main/utils/common.py

```diff
@@ -97,6 +97,14 @@ def to_python_boolean(value, allow_none=False):
         raise ValueError(_(u'Unable to convert "%s" to boolean') % six.text_type(value))
 
 
+def region_sorting(region):
+        if region[1].lower() == 'all':
+            return -1
+        elif region[1].lower().startswith('us'):
+            return 0
```

Comment:
This is why the rest of the world hates us, @matburt

Reply:
:us: :fireworks: